src/fswAlgorithms/sensorInterfaces/simpleInstrumentController/_UnitTest/test_simpleInstrumentController.py

```diff
@@ -90,6 +90,8 @@ def simpleInstrumentControllerTestFunction(show_plots):
 
     # Initialize the test module configuration data
     moduleConfig.attErrTolerance = 0.1                           # set the attitude error tolerance
+    moduleConfig.rateErrTolerance = 0.01                      # set the attitude rate error tolerance
+    moduleConfig.useRateTolerance = 1                            # enable attitude rate error tolerance
```

Comment:
You are only changing checking one case here were `moduleConfig.useRateTolerance` is set to 1.  Please check a case as well where this is set to zero, but the user specified a non-zero `moduleConfig.rateErrTolerance` value.  You can check all cases with a single python file using the purest parametrize properly.

Also, below you are only checking the case where `|omega|` is zero.  Setup a guidance case where it is non-zero.  Again, the pytest parameterize method makes this easy to check in one test.

Reply:
Added comprehensive tests to check a variety of success and failure cases relating to the rate threshold.